_posts/2022-02-15-finding-a-needle-in-a-haystack.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: Finding a needle in a haystack with git bisect
+author_avatar: https://avatars.githubusercontent.com/u/16843?v=4
+author_name: Steven Chau
+author_github_username: whereisciao
+excerpt: Finding a bad commit in an unfamiliar code base is like finding a needle in a haystack. Determined to find the root cause of the issue, I turned to an essential tool in my tool box, git bisect.
```

Comment:
I defined an `excerpt` as I preferred the image to be below the title tag, but not have the image included on the index page.